app/src/main/java/org/jellyfin/androidtv/util/DeviceUtils.java

```diff
@@ -25,36 +25,41 @@
     // Nvidia Shield TV Model
     private static final String SHIELD_TV_MODEL = "SHIELD Android TV";
 
+    static String getBuildModel() {
```

Comment:
function isn't marked as nullable, might also be better to use a default value (like "Unknown" or something like that) so we don't need to check nullability everywhere,